cmake/neodlrInstall.cmake

```diff
@@ -16,7 +16,7 @@ endforeach()
 
 set(NEODLR_HEADERS include/dlr.h)
 
-set_target_properties(neodlr PROPERTIES PUBLIC_HEADER "include/dlr.h;include/dlr_tvm.h;include/dlr_treelite.h")
+set_target_properties(neodlr PROPERTIES PUBLIC_HEADER ${NEODLR_HEADERS})
```

Comment:
Won't this break any client applications dep on these removed headers?

Reply:
we only depend on dlr.h